functions/public/Invoke-WPFMicrowin.ps1

```diff
@@ -117,14 +118,38 @@ public class PowerManagement {
 			if (Test-Path $driverPath)
 			{
 				Write-Host "Adding Windows Drivers image($scratchDir) drivers($driverPath) "
-				Add-WindowsDriver -Path "$scratchDir" -Recurse -Driver "$driverPath"
+				dism /English /image:$scratchDir /add-driver /driver:$driverPath /recurse | Out-Host
 			}
 			else 
 			{
 				Write-Host "Path to drivers is invalid continuing without driver injection"
 			}
 		}
 
+		if ($importDrivers)
+		{
+			Write-Host "Exporting drivers from active installation..."
+			if (Test-Path "$env:TEMP\DRV_EXPORT")
+			{
+				Remove-Item "$env:TEMP\DRV_EXPORT" -Recurse -Force
+			}
+			New-Item -Path "$env:TEMP\DRV_EXPORT" -ItemType Directory -Force
+			dism /english /online /export-driver /destination="$env:TEMP\DRV_EXPORT" | Out-Host
```

Comment:
Perhaps, export drivers to "inject drivers" directory? That way they will all be in once place and if you check the inject drivers checkbox they will get injected immediately?

Reply:
I've modified this function to take advantage of the injection directory if the option is enabled and the directory exists. Otherwise, it'll keep using `$env:TEMP\DRV_EXPORT`.

![powershell_ez0YJ1KGsv](https://github.com/ChrisTitusTech/winutil/assets/101426328/48783ad0-0cb0-4be9-a448-150c3ad96732)

![powershell_XVm4rjQeZU](https://github.com/ChrisTitusTech/winutil/assets/101426328/e77461b3-205b-44a2-b186-5c6c860a750b)

![powershell_1tGzyRJJs1](https://github.com/ChrisTitusTech/winutil/assets/101426328/dd6d0a29-610d-48bc-ae19-3f1faaef0d91)

The advantage is that, if a driver injection directory were to be specified, this function won't remove it afterward. I've tried this option but haven't determined if the default option still works. It should, but it's best to test it